src/wp-includes/comment-template.php

```diff
@@ -2646,6 +2648,7 @@ function comment_form( $args = array(), $post = null ) {
 		),
 		'comment_notes_after'  => '',
 		'action'               => site_url( '/wp-comments-post.php' ),
+		'novalidate_form'      => false,
```

Comment:
I'm wondering if this would be clearer as `novalidate_att`, and indicate that this is the behavior of the novalidate attribute. I'm on the fence about this, because we can't really avoid the double-negative. I'm not sure there's any foolproof way to communicate how the false state of a negative attribute is expressed.

Reply:
Good question. I picked this name to parallel to keys like `class_container`, `id_form`, and `name_submit` which is pattern is attribute name followed by the thing the attribute is for.

Reply:
It makes sense; it just doesn't follow the pattern used elsewhere in the same array. E.g. 'action' => "value of the action attribute".

Reply:
Consistency, apparently, is hard. ;)